# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sonypid/fixes.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

X11R6 is a relict of the past, X11R7 or better just X11 now ... also we need
.../lib64 on many architectures and last but not least gcc-4.1 warns about
missing sentinal due 0 vs. NULL.

  - Rene Rebe <rene@exactcode.de>

diff -u sonypid-1.9.1/Makefile sonypid-1.9.1-fixed/Makefile
--- sonypid-1.9.1/Makefile	2004-02-24 11:35:13.000000000 +0100
+++ sonypid-1.9.1-fixed/Makefile	2006-03-04 23:16:26.348888000 +0100
@@ -2,7 +2,7 @@
 CFLAGS = -Wall -Wstrict-prototypes -O2 -pipe
 SONYPIH = /usr/include/linux
 LDFLAGS =
-LIBS = -L/usr/X11R6/lib -lX11 -lXtst
+LIBS = `pkg-config x11 --cflags --libs` -lXtst
 
 PREFIX = /usr/local
 DESTBIN = $(PREFIX)/bin
Only in sonypid-1.9.1-fixed/: sonypi.h
Only in sonypid-1.9.1-fixed/: sonypid
diff -u sonypid-1.9.1/sonypid.c sonypid-1.9.1-fixed/sonypid.c
--- sonypid-1.9.1/sonypid.c	2004-02-24 11:35:13.000000000 +0100
+++ sonypid-1.9.1-fixed/sonypid.c	2006-03-04 23:15:56.855044750 +0100
@@ -202,7 +202,7 @@
 				break;
 			case SONYPI_EVENT_CAPTURE_PRESSED:
 				if (!fork()) {
-					execlp("xterm", "xterm", 0);
+					execlp("xterm", "xterm", NULL);
 					exit(0);
 				}
 				verboseEvent("Capture Pressed");
Only in sonypid-1.9.1-fixed/: sonypid.o
